episodes/4-minimal-reproducible-code.Rmd

```diff
@@ -128,6 +128,14 @@ counts_per_day <- krats %>%
   group_by(year, plot_id, plot_type, month, day, species_id) %>%
   summarize(count_per_day = n())
 
+### ***EDIT-start *** ###
+
+# --> Why doesn't this use "date" like we had set up earlier?
+# --> Is per day even the best way to do it? This isn't even average per day, just total per day.... Doesn't seem right
```

Comment:
Can you expand on what you mean by "doesn't seem right"? My goal here was to calculate the total number of individuals caught per day, so that then I could use boxplots to see the general range of how many individuals tended to be caught on a given day in each plot.